src/main/java/ua/chernonog/onlinebookstore/exception/EntitySaveException.java

```diff
@@ -0,0 +1,7 @@
+package ua.chernonog.onlinebookstore.exception;
+
+public class EntitySaveException extends RuntimeException {
+    public EntitySaveException(String message) {
+        super(message);
```

Comment:
```suggestion
    public EntitySaveException(String message, Throwable e) {
        super(message, e);
```
I think would be better pass e to this exception, if you use it only in try-catch of repository

Reply:
agree